interpreter.h

```diff
@@ -2,9 +2,20 @@
 #define __interpreter_h_
 
 #include <stdio.h>
-
```

Comment:
There should be a new line between the standard library includes and the others.